src/main/java/com/sprint/mission/discodeit/repository/file/FileUserRepository.java

```diff
@@ -0,0 +1,96 @@
+package com.sprint.mission.discodeit.repository.file;
+
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.repository.UserRepository;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.UUID;
+
+public class FileUserRepository implements UserRepository {
+    private final Path directory;
+
+    public FileUserRepository(String string) {
```

Comment:
사용하지 않는 인자는 제거해도 될 것 같아요.
```suggestion
    public FileUserRepository() {
```